symplyphysics/laws/hydro/hydraulic_press.py

```diff
@@ -0,0 +1,43 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output)
+
+# Description
+## If both vertically positioned cylinders of communicating vessels are closed with pistons, then with the help of external forces applied to the pistons,
+## a large pressure can be created in the liquid, many times exceeding the hydrostatic pressure at any point in the system.
+## If the pistons have different areas, then different forces act on them from the liquid side. The same modulus, but oppositely directed external forces must be applied to the pistons to keep the system in balance.
+
+## Law: F1 / S1 = F2 / S2

```

Comment:
I'd like to see the proof as soon as this PR is merged.

Reply:
https://www.webmath.ru/poleznoe/fizika_127_gidravlicheskij_press.php

Reply:
Yes, but I want to see it as a code.